pkgs/development/python-modules/chex/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, absl-py
+, dm-tree
+, jax
+, jaxlib
+, numpy
+, toolz
+}:
+
+buildPythonPackage rec {
+  pname = "chex";
+  # As of 2021-12-29, the latest official version has broken tests with jax 0.2.26.
+  version = "5adc10e0b4218f8ec775567fca38b68bbad42a3a";
```

Comment:
Version should either be unstable-yyyy-mm-dd with the revision only being used in the fetcher or alternatively you can use the format described here: https://github.com/NixOS/rfcs/pull/107

Reply:
Thanks for the link, fixed.